### PR TITLE
Typos and fixes

### PR DESCRIPTION
**Pull request by bentley**

### CODE REVIEW

File: src/epub/text/chapter-1.xhtml [hunk 1]
```diff
@@ -74,7 +74,7 @@
...
+			<p>On one side was a card label, tacked round the edges and addressed in a foreign handwriting: “<abbr>M.</abbr> Léon Felix, 141 West Jubb Street, Tottenham Court Road, London, <abbr epub:type="z3998:place">W</abbr>., via Rouen and long sea,” with the words “Statuary only” printed with a rubber stamp. The label bore also the sender’s name: “Dupierre <span xml:lang="fr">et <abbr>Cie.</abbr></span>, Fabricants de la Sculpture Monumentale, Rue Provence, Rue de la Convention, Grenelle, Paris.” Stencilled in black letters on the woodwork was “Return to” in French, English, and German, and the name of the same firm. Broughton examined the label with care, in the half-unconscious hope of discovering something from the handwriting. In this he was disappointed, but, as he held the hand-lamp close, he saw something else which interested him.</p>
```
Comment (acabal) [inline src/epub/text/chapter-1.xhtml:77]: This period is incorrect, W is a postal code in London

**Comment by acabal** [comment]
I think we need to undo 4db71bf as we don't change punctuation in that way. You can fix anything that is an actual transcription error, but we don't add or remove punctuation like that.

Additionally, you fixed a quote in `‘spots’` but `‘spots’` should actually be double quoted, not single quoted.

**Review by acabal — CHANGES_REQUESTED** [review]

**Comment by acabal** [comment]
I took a minute to fix these myself and merge everything in. Thanks!

**Comment by bentley** [comment]
Thanks, I got a bit distracted.

**Pull request closed without merging**